tests/Sinch.Tests/Core/HttpTests.cs

```diff
@@ -143,7 +143,7 @@ public async Task SendSinchHeader()
 
             var uri = new Uri("http://sinch.com/items");
 
-            var sdkVersion = new AssemblyName(typeof(Http).GetTypeInfo().Assembly.FullName).Version.ToString();
+            var sdkVersion = new AssemblyName(typeof(Http).GetTypeInfo()!.Assembly!.FullName!).Version!.ToString();
```

Comment:
I'm curious about this sudden change in the `HttpTests` regarding the sdkVersion

Reply:
To clear warnings